src/api.js

```diff
@@ -21,11 +21,13 @@ const post = (url, params) => {
   .then(res => res.json());
 }
 
-const importPost = (url, file, params) => {
+const postFormData = (url, params, file) => {
   const formData = new FormData();
-  formData.append('pass', params.pass);
-  formData.append('type', params.type);
-  formData.append('', file);
+
+  Object.keys(params).map(key => {
+    formData.append(key, params[key]);
+  })
```

Comment:
Very very not important, but I think it is more appropriate to use `forEach` here, because `map` is more so defined as a way of mutating one set of data to another.  This returns an array, whereas I think `forEach` makes more technical sense for looping through a set of data.  Anyway, only mentioning for educational purposes.